website/docs/tutorials/08-interacting-with-starknet/03-deploy-account.md

```diff
@@ -0,0 +1,17 @@
+# Creating account
+
+## Overview
+
+In order to create a new account, you need to deploy an account contract. StarkNet expects a certain interface from an account contract. [OpenZeppelin](https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/openzeppelin/account/presets/Account.cairo) and [ArgentX](https://github.com/argentlabs/argent-contracts-starknet/blob/develop/contracts/account/ArgentAccount.cairo) provide account contract implementations that you can use.
+
+There are two ways of deploying a new account contract:
+- sending `DEPLOY_ACCOUNT` transaction
+- using already deployed account contract and [UDC](https://community.starknet.io/t/universal-deployer-contract-proposal/1864)
+
+## Sending `DEPLOY_ACCOUNT` transaction 
+
+Protostar allows you to send the `DEPLOY_ACCOUNT` transaction from the CLI by calling the [`deploy-account` command](/docs/cli-reference#deploy-account).
+However, before you send such transaction you need to:
+1. find class hash of the account contract you want to be deployed and supports `DEPLOY_ACCOUNT` transaction
```

Comment:
How can I find it?

Reply:
No idea

Reply:
I think @Arcticae or @Solpatium can know. Adding this information is definitely a must here because people will ask us on Discord otherwise.

Reply:
I think it's the responsibility of the entity that declared the contract to make that information easily accessible.

Reply:
[explain how to find the class hash](https://github.com/software-mansion/protostar/pull/1105/commits/5aaab0c89c02292b87917430c3bbe0fecb2df387)